### ISSUE TITLE
Security fix: Update alpine base image to 3.7

### ISSUE THREAD

**Issue by usr42** [distribution/distribution-library-image#64]
The currently used alpine 3.4 base image has a known CVS/vulnerability.

Security fix for at least CVE-2016-8859 (see Scan result for registry:2.6.2)

See PR: https://github.com/docker/distribution-library-image/pull/62

**Comment by usr42** [comment]
The PR is already open pretty long. 
I've created a fixed build at quay.io: 
```
docker pull quay.io/usr42/registry:latest
```
You can get details about the build at https://quay.io/usr42/registry:latest

I've already created a build for alpine:3.7:
```
docker pull quay.io/usr42/registry:upgrade_alpine_3.7
```

**Comment by mlowicki** [comment]
I guess we should change title to "Security fix: Update alpine base image to 3.7".

**Comment by usr42** [comment]
@mlowicki Good point. I've changed the title.
Unfortunately, my feeling is, there will be no reaction either way 👎

**Comment by mlowicki** [comment]
@dmcgowan any thoughts?

**Comment by dmcgowan** [comment]
We update the alpine versions with releases of the registry. If you have found a vulnerability that effects the registry or the registry image, please email security@docker.com. This doesn't look like a CVE that has an impact on the registry, but I will defer to @tianon on what the policy is on updating base images for official based on security scan results.

**Comment by mlowicki** [comment]
@dmcgowan  @tianon maybe docker images should have explicit aliases to different version of Alpine so it would be something to choose an ease the migration?

**Comment by mlowicki** [comment]
@usr42  you can try https://github.com/operasoftware/docker-registry-image instead (https://hub.docker.com/r/operasoftware/registry/).

**Comment by milosgajdos** [comment]
Closing as wildly outdated. The latest available release is built of 3.20 https://github.com/distribution/distribution-library-image/blob/a943e89c3efe06134cd9a4b439203c5341082cbc/Dockerfile#L1
